keras/backend/mxnet_backend.py

```diff
@@ -17,6 +17,8 @@
 _REENTRY = False
 NAME_SCOPE_STACK = []
 
+py_all = all
```

Comment:
why do we need to define py_all here?

Reply:
following the convention in `tensorflow_backend` file